app/js/lib/simple_currency_formatter.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * This number formatter class is a primitive subset of the intl library ( https://www.npmjs.com/package/intl ),
+ * which is not available in all browsers and node versions.
+ */
+
+var parseGermanFloat = function ( amountStr ) {
```

Comment:
perhaps put the first var name on the next line as well, so its clear this is a pile of assigns, and not a single one?